---
title: '« Workfront Proof : erreur 500 lors de l’accès à Workfront Proof via l’API ou Workfront Fusion »'
description: '« Lorsqu’un utilisateur ou une utilisatrice accède à l’action getAllProofs de l’API Proof, le serveur Workfront Proof renvoie le message : erreur interne du serveur 500 »'
hidefromtoc: true
feature: Workfront Proof
source-git-commit: 98d56729e44e7ab47e201bdfc00db8d40c5f15f6
workflow-type: tm+mt
source-wordcount: '102'
ht-degree: 100%

---


# [!DNL Workfront Proof] : erreur 500 lors de l’accès au [!DNL Workfront Proof] via l’API ou [!DNL Workfront Fusion]

>[!NOTE]
>
>L’équipe produit évalue actuellement la résolution de ce problème, qui peut nécessiter des améliorations du produit. Les améliorations apportées aux produits sont communiquées dans les annonces de produit et non dans les mises à jour de maintenance.

<!--This article is on Proof and Fusion TOCs-->

Lorsqu’un utilisateur ou une utilisatrice accède à l’action [!UICONTROL `getAllProofs`] de l’API [!DNL Workfront Proof], le serveur renvoie le message suivant :

[!UICONTROL Erreur interne du serveur 500]

[!DNL Workfront Fusion] utilise l’API [!DNL Workfront Proof] pour les modules [!DNL Workfront Proof], cette erreur peut donc être renvoyée à un module, arrêtant un scénario.

_Premier signalement le 28 avril 2023._

